extensions/barrage/src/test/java/io/deephaven/extensions/barrage/chunk/BarrageColumnRoundTripTest.java

```diff
@@ -425,18 +425,18 @@ public void assertExpected(final WritableChunk<Attributes.Values> untypedOrigina
         final ChunkType chunkType = ChunkType.fromElementType(type);
 
         WritableChunk<Attributes.Values> rtData = null;
-        final WritableChunk<Attributes.Values> data = chunkType.makeWritableChunk(4096);
+        WritableChunk<Attributes.Values> data = chunkType.makeWritableChunk(4096);
```

Comment:
Isn't data going to leak, too? Why did it become non-final?

Reply:
That data chunk is passed off to the ChunkInputStreamGenerator as a matter of ownership. One line 432 that StreamGenerator is now correctly being tried with resources, which enables this chunk to be closed correctly. The StreamGenerator needs it for its entire life. 

Final vs non-final though was probably just an iterative change while Colin was looking for the source of the issue with this test's failure.

Reply:
OK. Thanks for the clarification on ownership.

Reply:
fixed final, and i made rtData more "local" so that it is easier to clean up